.github/workflows/tests-nightly.yml

```diff
@@ -3,7 +3,8 @@ on:
   workflow_dispatch: { }
   schedule:
   - cron: '0 5 * * *' # 5AM UTC = 12AM EST
-
+env:
+  PR_REVIEWERS: "ddietterich,yuhuyoyo,zloery,mikenev,melissachang"
```

Comment:
could we notify the on-call directly? Refer to chat on devops group. 
this way we dont need to maintain this list.

Reply:
actually we are already notifying slack about nightly outcome so I don't think sending email is necessary.